src/aal/aalmediaplayerservice.cpp

```diff
@@ -271,7 +272,7 @@ void AalMediaPlayerService::setMedia(const QUrl &url)
     if (m_mediaPlaylistProvider == nullptr || m_mediaPlaylistProvider->mediaCount() == 0)
     {
         // errors are delivered via Player::errorOccurred()
-        m_hubPlayerSession->openUri(url);
+        m_hubPlayerSession->openUri(url, headers);
```

Comment:
How does the callee deal with the new parameter?

Reply:
The parameter is optional by default but did exist already, as per this header file:
https://github.com/ubports/media-hub/blob/xenial/src/lib/MediaHub/player.h#L142